src/util/HttpServer/streamable_body.h

```diff
@@ -64,9 +65,11 @@ class streamable_body::writer {
    * conceptually can't allow const access.
    */
   template <bool isRequest, class Fields>
-  writer([[maybe_unused]] boost::beast::http::header<isRequest, Fields>& h,
-         value_type& b)
-      : _body{b} {}
+  writer(boost::beast::http::header<isRequest, Fields>& header, value_type& b)
```

Comment:
I suppose that the generic type alias `value_type` is a `boost` requirement.
Even if so this class is much more readable if the `_body` becomes a `_streamGenerator`/ `streamGenerator`.